### PR TITLE
[Chore] Change with Github Template

### PR DESCRIPTION
**Pull request by indraseptama**
In this PR, I modified the PR template file that was originally intended for GitLab to be used for GitHub. Additionally, I added CI for Dart code metrics.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+### Pre-Deployment Checklist
+
+- [ ] Run Local OK (Mandatory)
+- [ ] Run Dev Build on Real Device
+- [ ] Bilingual OK (Mandatory, for new product/feature or if there is a change to wording)
+
+### Test Checklist
+
+- [ ] Tested by QE
+- [ ] Tested by PM
```
Comment (yumnapratista24) [inline .github/PULL_REQUEST_TEMPLATE.md:29]: ndras ini tested by QE sm PM kyknya gak perlu deh (?) sama klo next kita mulai implement unit testing menurut lu gimana ndras ? @indraseptama

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+### How to Test
+
+Please include steps of how you tested this changes
+
+### Pre-Deployment Checklist
+
+- [ ] Run Local OK (Mandatory)
+- [ ] Run Dev Build on Real Device
+- [ ] Bilingual OK (Mandatory, for new product/feature or if there is a change to wording)
```
Comment (yumnapratista24) [inline .github/PULL_REQUEST_TEMPLATE.md:24]: Ini juga gak perlu broo

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (indraseptama) [reply .github/PULL_REQUEST_TEMPLATE.md:29]: Boleh broo

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (indraseptama) [reply .github/PULL_REQUEST_TEMPLATE.md:24]: Hokee, ini copas dari kangtor dulu wkwkw

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+### Background
+
+Clearly and concisely describe the feature including motivation, context and link of technical design document links, quip, screenshot, etc.
+
+If this a bugfix then describe the problem and explain the technical solution you have provided. Attach a document and screenshot if needed.
+If you make changes to the Amplitude then attach a screenshot or provide the amplitude url if needed.
+You can put additional notes here.
+
+### Impacted Products
```
Comment (yumnapratista24) [inline .github/PULL_REQUEST_TEMPLATE.md:9]: fggsgdsgdsgdsgdsgsdgds

Comment (indraseptama) [reply .github/PULL_REQUEST_TEMPLATE.md:9]: aneh

**Review by yumnapratista24 — APPROVED** [review]

**Pull request merged**